uniffi_bindgen/src/interface/mod.rs

```diff
@@ -129,7 +129,16 @@ impl ComponentInterface {
     }
 
     /// Add a metadata group to a `ComponentInterface`.
-    pub fn add_metadata(&mut self, group: uniffi_meta::MetadataGroup) -> Result<()> {
+    pub fn add_metadata(&mut self, mut group: uniffi_meta::MetadataGroup) -> Result<()> {
+        // Update namespace docstring if input is some, otherwise update
```

Comment:
This is the only sketchy thing in this PR. I didn't want to manually implement the required Eq, PartialEq traits, and this seemed to be the only place where Eq is used.